india_forecast_app/data/nwp.py

```diff
@@ -0,0 +1,64 @@
+import xarray as xr
+import os
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def regrid_nwp_data(nwp_ds: xr.Dataset, target_coords_path: str) -> xr.Dataset:
+    """This function loads the  NWP data, then regrids and saves it back out if the data is not
+    on the same grid as expected. The data is resaved in-place.
+
+    method can be 'conservative' or 'bilinear'
```

Comment:
What does this bit of the docstring refer to?

Reply:
oh, sorry, that can be removed now